src/components/calcite-fab/calcite-fab.stories.ts

```diff
@@ -0,0 +1,69 @@
+import { boolean, select, text, withKnobs } from "@storybook/addon-knobs";
+import {
+  Attributes,
+  createComponentHTML as create,
+  darkBackground,
+  parseReadme,
+  titlelessDocsPage
+} from "../../../.storybook/utils";
+import readme from "./readme.md";
+import { ATTRIBUTES } from "../../../.storybook/resources";
+import { ICONS } from "./resources";
+const { appearance, dir, scale, theme } = ATTRIBUTES;
+
+export default {
+  title: "components|calcite-fab",
+  decorators: [withKnobs],
+  parameters: {
+    backgrounds: darkBackground,
+    docs: {
+      page: titlelessDocsPage
+    },
+    notes: parseReadme(readme)
+  }
+};
+
+const createAttributes: () => Attributes = () => [
+  {
+    name: "appearance",
+    value: select("appearance", appearance.values, appearance.values[2])
+  },
+  {
+    name: "dir",
+    value: select("dir", dir.values, dir.defaultValue)
+  },
+  {
+    name: "disabled",
+    value: boolean("disabled", false)
+  },
+  {
+    name: "icon",
+    value: text("icon", ICONS.plus, "")
```

Comment:
Change to `value: text("icon", ICONS.plus)` and its good to go

Reply:
fixed